docs/indices.rst

```diff
@@ -144,5 +144,5 @@ Glossary
        `models from XSPEC
        <https://heasarc.gsfc.nasa.gov/xanadu/xspec/manual/XSappendixExternal.html>`_.
 
-       Sherpa can be built to use XSPEC versions 12.12.0, 12.11.1, 12.11.0,
-       12.10.1 (patch level `a` or later), 12.10.0, 12.9.1, or 12.9.0.
+       Sherpa can be built to use XSPEC versions 12.12.0, 12.11.1,
+       12.11.0, and 12.10.1 (patch level `a` or later).
```

Comment:
I'm impressed that you remembered to change the version number here, too. I suggest to change this to: The list of XSPEC versions that Sherpa can be build to use is detailed in <link to docs/install.rst here>. I don't think many many do that by themselves (most will use the CIAO build) so we don't have to maintain that list independent in several places.

Reply:
For future reference, I have decided to not make this change. It's a balance between DRY and a tolerable reading experience. As the number of versions go down the DRY-ness becomes less important (but when it goes up again we may want to implement the suggested change).